pb_plugins/templates/py/call.j2

```diff
@@ -1,16 +1,10 @@
 
-def {{ name }}(self):
+async def {{ name }}(self):
     """ Generated by dcsdkgen
 
-    :returns: Observable
+    :returns: Tuple[Success, Response]
```

Comment:
Since we are supporting only Python 3.6 and higher I am actually thinking about adding types in a later version - what do you think?
https://docs.python.org/3/library/typing.html